backend/config.py

```diff
@@ -18,7 +18,8 @@ class Settings(BaseSettings):
     CLIENT_SECRETS_FILE: str = "credentials.json"
     ENV: str = "dev"
     APP_URL: str
-    DATABASE_URL: str
+    DATABASE_URL_LOCAL: str
+    DATABASE_URL_DOCKER: str
```

Comment:
Can you leave a default string here so we don't need to add to .env file? I don't think it imposes a security risk if it's local. 

Side note What do you think about renaming?
```suggestion
    DATABASE_URL_LOCAL_VIRTUAL_ENV: str
    DATABASE_URL_LOCAL_DOCKER: str
```

Reply:
I changed this accordingly in config.py